src/schema/__tests__/ecommerce/create_order_with_artwork_mutation.test.js

```diff
@@ -11,54 +12,62 @@ describe("Create Order Mutation", () => {
     const resolvers = {
       Mutation: {
         createOrderWithArtwork: () => ({
-          order: exchangeOrderJSON,
-          errors: [],
+          orderOrError: { order: exchangeOrderJSON },
         }),
       },
+      OrderOrFailureUnion: {
+        __resolveType(obj, context, info) {
+          if (obj.order) {
+            return "OrderWithMutationSuccess"
+          } else if (obj.error) {
+            return "OrderWithMutationFailure"
+          }
+        },
+      },
```

Comment:
this took a bit of thinking to figure out our reported error, docs for [it are here](https://www.apollographql.com/docs/graphql-tools/resolvers#Unions-and-interfaces)